chrome/content/zotero/xpcom/data/feed.js

```diff
@@ -89,10 +112,22 @@ Zotero.defineProperty(Zotero.Feed.prototype, 'isFeed', {
 Zotero.defineProperty(Zotero.Feed.prototype, 'libraryTypes', {
 	value: Object.freeze(Zotero.Feed._super.prototype.libraryTypes.concat(['feed']))
 });
+Zotero.defineProperty(Zotero.Feed.prototype, 'unreadCount', {
+	get: function() this._feedUnreadCount
+});
+Zotero.defineProperty(Zotero.Feed.prototype, 'updating', {
+	get: function() this._updating,
+	set: function(v) {
+		if (!v == !this._updating) return; // Unchanged
+		this._updating = !!v;
+		Zotero.Notifier.trigger('statusChanged', 'feed', this.id);
+	}
```

Comment:
We should make this a getter instead of getter/setter. The getter should return `!!this._updating`. Then use `this._updating` to store the promise for the update. See `._updateFeed` below for more on this